en/patterns/av.md

```diff
@@ -13,6 +13,12 @@ Use the AV Pattern to complement a represenatation of an audio or video track wi
 
 The AV Pattern comes with the styling flexibility provided by the Icon Buttons and Linear Progress Bar that shape its layout.
 
+> [!WARNING]
+> After inserting an AV layout, you should trigger `Detach from Symbol` to break it up to the Components that are used to create the Pattern in order to be able to generate it as Angular code.
```

Comment:
I'd recommend either "to break it up into the Components" or "to break it down to the Components".

Reply:
@StefanIvanov The other changes you made look good, but I don't see changes for this type of warning in the files.  It's a minor grammar issue (it sounds weird) but if you don't think it's important enough to fix, I can approve the PR.  Or are you planning to make more updates and then do a find/replace on this?